packages/astro-rss/README.md

```diff
@@ -202,6 +185,112 @@ export const get = () => rss({
 });
 ```
 
+## `RSSFeedItem`
+
+An `RSSFeedItem` is a single item in the list of items in your feed. It represents a story, with `link`, `title` and `pubDate` fields. There are further optional fields defined below. You can also check the definitions for the fields in the [RSS spec](https://validator.w3.org/feed/docs/rss2.html#ltpubdategtSubelementOfLtitemgt).
+
+An example feed item might look like:
+
+```js
+const item = {
+  title: "Alpha Centauri: so close you can touch it",
+  link: "/blog/alpha-centuari",
+  pubDate: new Date("2023-06-04"),
+  description: "Alpha Centauri is a triple star system, containing Proxima Centauri, the closest star to our sun at only 4.24 light-years away.",
+  categories: ["stars", "space"]
+}
+```
+
+### `title`
+
+Type: `string (required)`
+
+The `<title>` attribute of the item in the feed.
+
+### `link`
+
+Type: `string (required)`
+
+The `<link>` attribute of the item in the feed containing the URL of the item on the web.
+
+### `pubDate`
+
+Type: `Date (required)`
+
+Indicates when the item was published.
+
+### `description`
+
+Type: `string (optional)`
+
+If the item is complete within itself, that is you are publishing the full content of the item in the feed, the `description` field may contain the full text (entity-encoded HTML is permitted). If the item is a stub, then the `description` may contain a synopsis of the item.
+
+### `content`
+
+Type: `string (optional)`
+
+If you want to supply both a short description and also the full content in an item, set the `content` field to the full, encoded text. See the [recommendations from the RSS spec for how to use and differentiate between `description` and `content`](https://www.rssboard.org/rss-profile#namespace-elements-content-encoded).
+
+### `categories`
+
+Type: `string[] (optional)`
+
+If you use tags or categories to categorize your content, you can add them as the `categories` field. They will be output as multiple `<category>` elements.
+
+### `author`
+
+Type: `string (optional)`
+
+Useful for multi-author blogs, the `author` field provides the email address of the person who wrote the item.
+
+### `commentsUrl`
+
+Type: `string (optional)`
+
+The `commentsUrl` defines a URL of a web page that contains comments on the item.
+
+### `source`
+
+Type: `object (optional)`
+
+Items that are republished from other publications may define a `source` which defines the `title` and `url` of the original feed in which it was published.
```

Comment:
``````suggestion
An object that defines the `title` and `url` of the original feed for items that have been republished from another source. Both are required propeties of `source` for proper attribution. 

```js
const item = {
	title: "Alpha Centauri: so close you can touch it",
  link: "/blog/alpha-centuari",
  pubDate: new Date("2023-06-04"),
	source:
		{
			title: "The Galactic Times",
			url: "https://galactictimes.space/feed.xml"
		}
}
```
``````
I also think an example (choose one that makes sense, formatted properly!) would make sense here.